crates/ty_python_semantic/resources/mdtest/snapshots/mro.md_-_Method_Resolution_Or…_-_`__bases__`_lists_wi…_(ea7ebc83ec359b54).snap

```diff
@@ -141,7 +141,7 @@ info[revealed-type]: Revealed type
 ```
 
 ```
-error[duplicate-base]: Duplicate base class `Spam`
+error[duplicate-base]: Duplicate base class `Eggs`
```

Comment:
This change is a little odd, maybe something to do with the ID generation affecting some sort order?

Reply:
Something was non-deterministic here before, so it's not related to your changes.

Reply:
I switched to using an `IndexMap` in https://github.com/astral-sh/ruff/commit/cb04343b3b5e7a8a0841c73537733fa5aac482a2 to try to improve the determinism of the order of diagnostics -- can you try rebasing and see if the order still changes after the rebase?

Reply:
Looks like that fixed it, thanks!

Reply:
Glad to hear it 😃